docker/Dockerfile.linux.amd64

```diff
@@ -1,9 +1,12 @@
-FROM plugins/base:multiarch
+FROM alpine:3.12
 
```

Comment:
the original `plugins/base:multiarch` ensures `ca-certificates` is included in the base image and it also sets some default environment variables such as `ENV GODEBUG netdns=go` which forces Go to use internal networking, which is important because otherwise it relies on a GLIBC which won't exist in the alpine image.

Reply:
Fixed